### PR TITLE
chore: shallow type fixes

### PR DESCRIPTION
**Pull request by mdrocco**
Following @stefanberger's suggestion, I am splitting #1139 into smaller PRs. This one provides "shallow" fixes, in the sense they should not change the behavior of the code at all:

- uninitialized variables
- variables initialized with wrong types
- missing/incomplete imports
- unspecified exceptions
- missing asserts (e.g., not None, instanceof, callable)
- missing type annotations

### CODE REVIEW

File: keylime/cli/options.py [hunk 1]
```diff
@@ -29,4 +31,4 @@ def get_opts_error(args):
         return True, "--allowlist-sig-url must also have --allowlist-sig-key"
     if args.allowlist_sig_key and not (args.allowlist_sig or args.allowlist_sig_url):
         return True, "--allowlist-sig-key must have either --allowlist-sig or --allowlist-sig-url"
-    return False, None
+    return False, ""
```
Comment (mpeters) [inline keylime/cli/options.py:34]: Why go with an empty string vs None here?

**Review by mpeters — CHANGES_REQUESTED** [review]

File: keylime/cli/options.py (hunk 1 above)
Comment (aplanas) [reply keylime/cli/options.py:34]: Because of the annotated type (`-> Tuple[bool, str]`)

**Review by aplanas — APPROVED** [review]

**Comment by aplanas** [comment]
Thanks, this is so much easy to review!

**Comment by mdrocco** [comment]
@aplanas that is also because it is a proper subset of #1139, that you probably reviewed entirely :)

File: keylime/api_version.py [hunk 2]
```diff
@@ -42,19 +42,23 @@ def normalize_version(v):
...
+    if "." not in base_version:
         latest_minor = latest_minor_version(base_version)
         if latest_minor != "0":
             return latest_minor
     return base_version
 
 
 def major(v):
-    return version.parse(str(v)).major
+    version_ = version.parse(str(v))
```
[the remark this answers is not in the document]
Comment (mdrocco) [reply keylime/api_version.py]: done

File: keylime/db/keylime_db.py [hunk 3]
```diff
@@ -96,8 +96,8 @@ def make_session(self, engine):
         To use: session = self.make_session(engine)
         """
         self.engine = engine
+        Session = scoped_session(sessionmaker())
```
[the remark this answers is not in the document]
Comment (mdrocco) [reply keylime/db/keylime_db.py:99]: because it is referenced outside the block.

**Review by aplanas — APPROVED** [review]

**Review by mpeters — APPROVED** [review]

**Pull request merged**